test/test_scripts_run_task.py

```diff
@@ -354,13 +356,14 @@ def _commit_file(message, filename):
 
 
 @pytest.mark.parametrize(
-    "base_ref,ref,files,hash_key",
+    "base_ref,ref,files,hash_key,shallow",
     [
-        (None, None, ["mainfile"], "main"),
-        (None, "main", ["mainfile"], "main"),
-        (None, "mybranch", ["mainfile", "branchfile"], "branch"),
-        ("main", "main", ["mainfile"], "main"),
-        ("main", "mybranch", ["mainfile", "branchfile"], "branch"),
+        (None, None, ["mainfile"], "main", True),
+        (None, "main", ["mainfile"], "main", False),
+        (None, "mybranch", ["mainfile", "branchfile"], "branch", True),
+        ("main", "main", ["mainfile"], "main", False),
+        ("main", "mybranch", ["mainfile", "branchfile"], "branch", True),
+        ("main", "mybranch", ["mainfile", "branchfile"], "branch", False),
```

Comment:
Let's keep the parameter set to `False` for all the existing tests, and then add new one where it is `True`. I wouldn't want to accidentally change the intent of one of the original tests.